dietpi/dietpi-software

```diff
@@ -14180,10 +14186,10 @@ _EOF_
 
 			fi
 			# - Lighttpd
-			if [[ -d '/etc/lighttpd/mods-available' ]]; then
+			if [[ -d '/etc/lighttpd/conf-available' ]]; then
```

Comment:
Good find 👍.

`lighty-disable-mod` fails with exit code when the config is disabled already. So to be sure, and to allow printing error messages (removing `2> /dev/null`) and principally error handling that call, we can check for the enabled config explicitly:
```
[[ -f '/etc/lighttpd/conf-enabled/99-dietpi-pihole.conf' ]] && G_EXEC_NOEXIT=1 G_EXEC lighty-disable-mod dietpi-pihole
[[ -f '/etc/lighttpd/conf-enabled/99-dietpi-pihole-block_public_admin.conf' ]] && G_EXEC_NOEXIT=1 G_EXEC lighty-disable-mod dietpi-pihole-block_public_admin
G_EXEC_NOEXIT=1 G_EXEC rm -f /etc/lighttpd/conf-{available,enabled}/99-dietpi-pihole{,-block_public_admin}.conf
```
Actually... `lighty-disable-mod` does nothing else than removing the symlink, so the last command covers the first two. Usually I like to use the CLI, to be sure that all is done correctly, but in this case it is very trivial, and if something did change in Lighttpd/with these commands, then we'd need to change a lot of scripts anyway. What do you think? Keeping it simple and remove configs without relying on those CLI commands?
```
G_EXEC_NOEXIT=1 G_EXEC rm -f /etc/lighttpd/conf-{available,enabled}/99-dietpi-pihole{,-block_public_admin}.conf
```

Reply:
I mean it's two configs only. But yes the more I think about it the less I see a reason to go the complicated way. The last command will remove both configs, enabled symlink and available file, regardless if Lighttpd binaries are actually still installed or only the /etc/lighttpd directory structure left, so that is all we need. With `-f` it ignores missing files without error out, so no need to check for each file individually.